includes/Classifai/Plugin.php

```diff
@@ -78,6 +78,7 @@ public function init_services() {
 		$this->services = [
 			'service_manager'     => new Services\ServicesManager( $classifai_services ),
 			'admin_notifications' => new Admin\Notifications(),
```

Comment:
Putting non-service-related classes in `$this->services` doesn't seem quite right, but I followed what was already there.

Reply:
@johnwatkins0 this was a side-effect of the refactor work done to allow the plugin to support multiple services. At the time, there wasn't a need to handle non-services differently. This can probably be added now as we have both notices and debug information .